### PR TITLE
Selftest

### PR DESCRIPTION
**Pull request by Salem-Luos**

### CODE REVIEW

**Comment by Salem-Luos** [comment]
Maybe use ctx.lock at the end of the test to checkout if timeout work

File: src/luos.c [hunk 1]
```diff
@@ -60,6 +60,20 @@ void Luos_Init(void)
...
+        while (package_index < package_number)
+        {
+            package_table[package_index].Init();
+            package_index += 1;
+        }
+    }
+    else
+    {
+        Luos_EmptyNode();
+    }
```
Comment (nicolas-rabault) [inline src/luos.c:76]: Is those modifications have any relation with self-test?

**Review by nicolas-rabault — CHANGES_REQUESTED** [review]
Why there is a package thing here?

File: src/luos.c (hunk 1 above)
Comment (ncros3) [reply src/luos.c]: Commits on luos.c had been dropped, those changes were a mistake.

**Review by ncros3 — APPROVED** [review]
That's OK for me.

**Pull request closed without merging**